six/modules/python/six.sicd/source/six_sicd.i

```diff
@@ -199,6 +221,17 @@ void writeNITF(const std::string& pathname, const std::vector<std::string>&
 Data* readNITF(const std::string& pathname,
         const std::vector<std::string>& schemaPaths);
 
+void crop_SICD(const std::string& inPathname,
```

Comment:
Why is this here?  Doesn't seem to do anything..?

Reply:
I can't directly call cropSICD from CropUtils.h because this line needs to be called first.
```
six::XMLControlFactory::getInstance().addCreator(
          six::DataType::COMPLEX,
	  new six::XMLControlCreatorT<six::sicd::ComplexXMLControl>());
```

So I made a wrapper called crop_SICD to call that line first.  This line is to tell swig to wrap that function.